src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -221,6 +224,9 @@ public function getInternalResponse(): Response
      * The origin response is the response instance that is returned
      * by the code that handles requests.
      *
+     * @phpstan-return TResponse
+     * @psalm-return TResponse
```

Comment:
We don't need both annotations. Let's only use the Psalm one. PHPStan will read that one as well.

```suggestion
     * @psalm-return TResponse
```

Reply:
done, and `@phpstan-param` removed as well